test/example-types/metricGroups/pivot.js

```diff
@@ -2282,6 +2412,283 @@ describe('pivot', () => {
+          '2015-10-01T00:00:00.000Z',
+        ],
+      },
+      {
+        drilldown: ['2016-01-01T00:00:00.000Z'],
+        skip: [],
+        include: [
+          '2016-01-01T00:00:00.000Z',
+          '2016-04-01T00:00:00.000Z',
+          '2016-07-01T00:00:00.000Z',
+          '2016-10-01T00:00:00.000Z',
+        ],
+      },
+    ]
+    let expandedRows = [
+      {
+        drilldown: [],
+        include: [
+          'District of Columbia',
+          'California',
+          'New York',
+          'Virginia',
+          'Texas',
+          'Massachusetts',
+          'Illinois',
+          'New Jersey',
+          'Ohio',
+          'Florida',
+        ],
+      },
+    ]
+    let input = {
+      key: 'test',
+      type: 'pivot',
+      values: [{ type: 'sum', field: 'LineItem.TotalPrice' }],
+      rows,
+      columns,
+      pagination: {
+        columns: {
+          drilldown: [],
+          skip: [],
+          expanded: expandedColumns,
+        },
+        rows: {
+          drilldown: ['New York'],
+          skip: [],
+          expanded: expandedRows,
+        },
+      },
+    }
+    let expectedIntial = {
+      key: 'test',
+      type: 'pivot',
+      values: [{ type: 'sum', field: 'LineItem.TotalPrice' }],
+      rows,
+      columns,
+      pagination: {
+        columns: {
+          drilldown: [],
+          skip: [],
+          include: [
+            '2015-01-01T00:00:00.000Z',
+            '2016-01-01T00:00:00.000Z',
+            '2017-01-01T00:00:00.000Z',
+            '2018-01-01T00:00:00.000Z',
+            '2019-01-01T00:00:00.000Z',
+            '2020-01-01T00:00:00.000Z',
+            '2021-01-01T00:00:00.000Z',
+            '2022-01-01T00:00:00.000Z',
+          ],
+          expanded: expandedColumns,
+        },
+        rows: {
+          drilldown: ['New York'],
+          skip: [],
+          expanded: expandedRows,
+        },
+      },
+    }
+
+    let { initial, makePages } = mapExpandedPages(input)
+    // console.log(JSON.stringify(initial))
+    expect(initial).to.eql(expectedIntial)
+
+    let includeValues = [
+      'New York',
+      'Albany',
+      'Brooklyn',
+      'Long Island City',
+      'Corona',
+      'Buffalo',
+      'Syracuse',
+      'Cooperstown',
+      'Rochester',
+      'White Plains',
+    ]
+    let expectedPages = [
+      {
+        key: 'test',
+        type: 'pivot',
+        values: [{ type: 'sum', field: 'LineItem.TotalPrice' }],
+        rows,
+        columns,
+        pagination: {
+          columns: {
+            drilldown: ['2015-01-01T00:00:00.000Z'],
+            skip: [],
+            include: [
+              '2015-01-01T00:00:00.000Z',
+              '2015-04-01T00:00:00.000Z',
+              '2015-07-01T00:00:00.000Z',
+              '2015-10-01T00:00:00.000Z',
+            ],
+            expanded: expandedColumns,
+          },
+          rows: {
+            drilldown: ['New York'],
+            skip: [],
+            include: [
```

Comment:
Is this the same result as the `includeValues` array above?  If so it would make it a little easier to understand the test if you just reused the variable here so the reader knows the value hasn't changed.

Reply:
Done